boards/wch/ch32v003evt/board.yml

```
@@ -0,0 +1,5 @@
+board:
+  name: wch_ch32v003evt
```

```suggestion
  name: ch32v003evt
```

It's better to avoid having vendor name in the board name (and yeah, I appreciate that means you need to change this in several places...). Interestingly you've already named the board's folder like that so... :)

---

@kholia I'll send yo a PR. It seems that boards/ is inconsistent with many boards including the vendor in the board name, however `git log` shows that the more recent ones don't.

---

@kholia see https://github.com/nzmichaelh/zephyr/commit/a802f99e9d9b82962034b5c492e1755437d99258

---

Applied and pushed - thanks!